plugins/inputs/systemd_units/shared_test.go

```diff
@@ -0,0 +1,82 @@
+package systemd_units
```

Comment:
Thanks for some additional tests, however let's keep the general tests in `systemd_units_test.go` please and remove this file.